__tests__/components/Wrapper.test.js

```diff
@@ -0,0 +1,66 @@
+jest.setTimeout(30000);
+import React from 'react';
+import { View } from 'react-native';
+import renderer from 'react-test-renderer';
+
+import {
+  Wrapper,
+  WrapperHorizontal,
+  WrapperVertical,
+  WrapperLandscape,
+  WrapperRow,
+  WrapperWrap,
+  InfoBox,
+  WrapperWithOrientation
+} from '../../src/components';
+
+describe('testing Wrapper style component', () => {
+  it('renders a default Wrapper', async () => {
+    const tree = renderer.create(<Wrapper />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders WrapperHorizontal', () => {
+    const tree = renderer.create(<WrapperHorizontal big />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders WrapperVertical', async () => {
+    const tree = renderer.create(<WrapperVertical />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders WrapperLandscape', async () => {
+    const tree = renderer.create(<WrapperLandscape noFlex />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders WrapperRow', async () => {
+    const tree = renderer.create(<WrapperRow center />);
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders WrapperRow', async () => {
```

Comment:
when you read the output of jest there are two test cases with the same name here. it... should not be identical. it should always be a string that describes the test and the two wrapper row tests are different.

Reply:
Correction made in 61c542c168e0cf5d5715f440ab76cfabfb8ccf7e